src/Desktop.Analyzers/Desktop.Analyzers.md

```diff
@@ -48,6 +48,16 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182350.aspx](http://msdn.microsoft.com/library/ms182350.aspx)
```

Comment:
I am indeed referring to the rule description (the paragraph beginning "Using XmlTextReader.Load(), creating...". And yes, I am suggesting we backtick-quote all the types or other code fragment in the descriptions. Could you clarify what you mean when you say "the purpose of the string is not for creating markdown file"? This _is_ a Markdown file, so I thought would be nice to take advantage of Markdown formatting where we can. Perhaps I'm misunderstanding you.